DQM/SiStripMonitorClient/plugins/SiStripDcsInfo.cc

```diff
@@ -1,20 +1,90 @@
+#include "DQM/SiStripCommon/interface/SiStripFolderOrganizer.h"
+#include "DQM/SiStripMonitorClient/interface/SiStripUtility.h"
+#include "DQMServices/Core/interface/DQMStore.h"
+#include "CalibFormats/SiStripObjects/interface/SiStripDetCabling.h"
+#include "CalibTracker/Records/interface/SiStripDetCablingRcd.h"
+#include "CondFormats/RunInfo/interface/RunInfo.h"
+#include "CondFormats/DataRecord/interface/RunSummaryRcd.h"
+#include "CondFormats/DataRecord/interface/SiStripCondDataRecords.h"
+#include "CondFormats/SiStripObjects/interface/SiStripDetVOff.h"
+#include "FWCore/Framework/interface/one/EDAnalyzer.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Framework/interface/ESWatcher.h"
+#include "FWCore/Framework/interface/LuminosityBlock.h"
+#include "FWCore/Framework/interface/Run.h"
+#include "FWCore/Framework/interface/EventSetup.h"
 #include "FWCore/MessageLogger/interface/MessageLogger.h"
 #include "FWCore/ServiceRegistry/interface/Service.h"
 #include "FWCore/ParameterSet/interface/ParameterSet.h"
-
-#include "DQMServices/Core/interface/DQMStore.h"
-#include "CondFormats/RunInfo/interface/RunInfo.h"
-
-#include "DQM/SiStripCommon/interface/SiStripFolderOrganizer.h"
-#include "SiStripDcsInfo.h"
-#include "DQM/SiStripMonitorClient/interface/SiStripUtility.h"
+#include "Geometry/Records/interface/TrackerTopologyRcd.h"
 
 #include <iostream>
 #include <iomanip>
 #include <cstdio>
-#include <string>
 #include <sstream>
 #include <cmath>
+#include <fstream>
+#include <string>
+#include <vector>
+#include <map>
+#include <unordered_map>
+
+class SiStripDetVOff;
+class SiStripDetCabling;
+class RunInfo;
+
+class SiStripDcsInfo : public edm::one::EDAnalyzer<> {
```

Comment:
Is this the right fix?
I think this should be moved to `DQMEDHarvester` instead.